olive/systems/python_environment/python_environment_system.py

```diff
@@ -82,9 +68,40 @@ def __init__(
         self.available_eps = None
 
         # path to inference script
-        self.inference_path = Path(__file__).parent.resolve() / "inference_runner.py"
-        self.pass_path = Path(__file__).parent.resolve() / "pass_runner.py"
-        self.device = self.accelerators[0] if self.accelerators else Device.CPU
+        parent_dir = Path(__file__).parent.resolve()
+        self.pass_runner_path = parent_dir / "pass_runner.py"
+        self.evaluation_runner_path = parent_dir / "evaluation_runner.py"
+
+    def _run_command(self, script_path: Path, config_jsons: Dict[str, Any], **kwargs) -> Dict[str, Any]:
+        """Run a script with the given config jsons and return the output json."""
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            tmp_dir_path = Path(tmp_dir).resolve()
+
+            # write config jsons to files
+            args = []
+            for key, config_json in config_jsons.items():
+                config_json_path = tmp_dir_path / f"{key}.json"
+                with config_json_path.open("w") as f:
+                    json.dump(config_json, f, indent=4)
+                args.append(f"--{key} {config_json_path}")
+
+            # command to run
+            command = f"python {script_path} {' '.join(args)}"
```

Comment:
nit: I think using string list then join is similar to C# StringBuilder, which should have better performance than string concatenation

Reply:
Changed to use a string list for `command`